js/gateio.js

```diff
@@ -649,13 +650,13 @@ module.exports = class gateio extends Exchange {
                         'swap': swap,
                         'option': option,
                         'derivative': true,
-                        'contract': true,
+                        'contract': (contractSize !== '1'),
```

Comment:
Whats the logic behind this?

Reply:
@frosty00 I would agree with Sam – I think the edit in this line is wrong.

Reply:
I was actually asking a question because I didn't know, so there's nothing to agree with me here